iai_apartment/launch/kitchen2_display.launch

```diff
@@ -0,0 +1,17 @@
+<launch>
+
```

Comment:
There should be a static transform publisher, which links the apartment to map

    <node pkg="tf"
        type="static_transform_publisher"
        name="iai_apartment_room_link_broadcaster" 
        args="0.0 0.0 0. 0.0 0. 0. map iai_apartment/kitchen_root 25">
    </node>